### ISSUE TITLE
"esy run" issue after building the app on Ubuntu 20.04

### ISSUE THREAD

**Issue by Lap1n** [onivim/oni2#2460]
Hi,

I've successfully built the project on Ubuntu 20.04.1 LTS  but when I run the app with "esy run", it doesn't launch.

I'm getting the following error: 
```
[TRACE]    +0ms Revery.UI.Render : BEGIN: Render frame
[DEBUG]    +0ms Revery.CanvasContext : Unable to create native interface. Falling back to SDL2...
[INFO]     +0ms Revery.CanvasContext : Creating Skia context...
[ERROR]    +0ms Revery.CanvasContext : Unable to create skia context
[TRACE]    +0ms Revery.UI.Render : END: Render frame
```

It seems related to https://github.com/revery-ui/revery/issues/993

The full log ouput is attached.
[output.txt](https://github.com/onivim/oni2/files/5240116/output.txt)

**Comment by bryphe** [comment]
Hmm, is this running via a VM? It seems like revery/Onivim is having trouble finding an OpenGL device - from the logs:

```
[30m[41m[ERROR][0;30m[0m    +0ms [93mRevery.SDL2[0m : SDL_Init failed: No available video device
...
[30m[44m[INFO] [0;30m[0m    +0ms [31mRevery.Core.Window[0m : OpenGL hardware info:
[30m[44m[INFO] [0;30m[0m    +0ms [31mRevery.Core.Window[0m :   renderer: Unable to get OpenGL proc address for glGetString
[30m[44m[INFO] [0;30m[0m    +0ms [31mRevery.Core.Window[0m :   version: Unable to get OpenGL proc address for glGetString
[30m[44m[INFO] [0;30m[0m    +0ms [31mRevery.Core.Window[0m :   vendor: Unable to get OpenGL proc address for glGetString
[30m[44m[INFO] [0;30m[0m    +0ms [31mRevery.Core.Window[0m :   shadingLanguageVersion: Unable to get OpenGL proc address for glGetString
```

I'm curious also if the `glxinfo` command is available and what it shows.

**Comment by Lap1n** [comment]
No, I'm not running it from a VM. 

The logs from glxinfo is attached.
[glxinfo.txt](https://github.com/onivim/oni2/files/5241308/glxinfo.txt)

**Comment by bryphe** [comment]
Good catch on being related to - revery-ui/revery#993 - I believe that is likely the case, given that `glxinfo` looks good and your OpenGL drivers are in good shape.

The TL;DR is that likely the X11 development libraries / headers weren't available when SDL2 first built - in that case, SDL2 doesn't have a default device. I put some more details + steps to fix in revery-ui/revery#993.

I'll resolve this as external since it's tracked via: revery-ui/revery#993

Would also be an interesting data point to see if the project builds for you.
